toolbox

```diff
@@ -347,6 +347,17 @@ create_environment_options()
 }
 
 
+set_terminal_size()
+{
+    stty size|if read -r lines cols; then
```

Comment:
Umm... why the `if`? I guess you wanted to handle cases where the `stty` invocation might have failed?

If so, then it would be more readable to move it out into a separate `if` branch like we do for other commands; and then use `cut` to extract the size fields because `read` is a bit weird to use when there's no conditional branch or loop to go with it.

Reply:
You're right. okay, let me try as you suggested.